src/hb-subset-input.cc

```diff
@@ -204,3 +215,29 @@ hb_subset_input_get_retain_gids (hb_subset_input_t *subset_input)
 {
   return subset_input->retain_gids;
 }
+
+HB_EXTERN void
+hb_subset_input_set_name_legacy (hb_subset_input_t *subset_input,
+				 hb_bool_t name_legacy)
+{
+  subset_input->name_legacy = name_legacy;
+}
+
+HB_EXTERN hb_bool_t
+hb_subset_input_get_name_legacy (hb_subset_input_t *subset_input)
+{
+  return subset_input->name_legacy;
+}
+
+HB_EXTERN void
+hb_subset_input_set_obfuscate_names (hb_subset_input_t *subset_input,
+				     hb_bool_t obfuscate_names)
+{
```

Comment:
I don't see any code in hb-ot-name-table.hh that uses obfuscate_names. If this PR isn't adding the obfuscate names implementation then I'd leave it out of the subset api and hb-subset util options for now.